pkg/install/install_test.go

```diff
@@ -58,35 +58,27 @@ func TestFillInTemplatesAllParameters(t *testing.T) {
 		GitReadOnly:        false,
 		ManifestGeneration: true,
 		AdditionalFluxArgs: []string{"arg1=foo", "arg2=bar"},
-		RegistryScanning:   true,
 		AddSecurityContext: true,
 	})
 }
 
 func TestFillInTemplatesMissingValues(t *testing.T) {
 	testFillInTemplates(t, 5, TemplateParameters{
-		GitURL:           "git@github.com:fluxcd/flux-get-started",
-		GitBranch:        "branch",
-		GitPaths:         []string{},
-		GitLabel:         "label",
-		RegistryScanning: true,
+		GitURL:                  "git@github.com:fluxcd/flux-get-started",
+		GitBranch:               "branch",
+		GitPaths:                []string{},
+		GitLabel:                "label",
+		RegistryDisableScanning: true,
```

Comment:
Presumably, this should now omit memcached deployment, and thereby have fewer manifests?

Reply:
Indeed:

```
(cd pkg/install; go test -v .)
```

fails.

Reply:
Fixed. I really need to fix the install package testing